rtv/terminal.py

```diff
@@ -599,7 +599,10 @@ def open_url_silent(url):
                     try:
                         os.dup2(null, 1)
                         os.dup2(null, 2)
-                        webbrowser.open_new_tab(url)
+                        if self.config['force_new_browser_window']:
```

Comment:
There's another place that ``webbrowser.open_new_tab()`` is called at the bottom of this method. That will also need to be updated for consistency.

Reply:
Oh ok I actually I didn't add it there on purpose because I thought that it doesn't make sense to do this for a console browser. I will add it.

Reply:
No worries. There are actually edge cases where that code path gets executed with GUI browsers too (like if the user's ``$DISPLAY`` is set wrong).